L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -38,11 +38,11 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         // Swipe to the second screen
         app.buttons["\(rootA11yId)PrimaryButton"].tap()
         currentScreen += 1
-        waitForExistence(app.buttons["\(rootA11yId)PrimaryButton"])
+        waitForExistence(app.buttons["SignSyncCardSecondaryButton"])
         snapshot("Onboarding-2")
 
         // Swipe to the third screen
```

Comment:
Whe I opened the issue there were only two onboarding cards, not sure we need this part (and if it is working) unless there is a new card introduced that I'm not aware of 🙈